src/sentry/api/serializers/models/organization.py

```diff
@@ -42,6 +42,7 @@
     ATTACHMENTS_ROLE_DEFAULT,
     DEBUG_FILES_ROLE_DEFAULT,
     EVENTS_MEMBER_ADMIN_DEFAULT,
+    GITHUB_OPEN_PR_BOT_DEFAULT,
```

Comment:
let's condense the `GITHUB_OPEN_PR_BOT_DEFAULT` and `GITHUB_PR_BOT_DEFAULT` into a single `COMMENT_BOT_DEFAULT` that's always true